### ISSUE TITLE
SRW - support import of virtual beamlines with fiber

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#291]
A version of CHX virtual beamline with the fiber should be able to be imported in Sirepo. Currently the importer complains about it:
```python
../sirepo/importer.py:134:import_python Error: Import failed: error unknown; exception=Element type <cyl_fiber> does not exist.; script=/home/vagrant/src/radiasoft/sirepo/run/user/XgcK3MP3/tmp/RifucQqL/in.py; filename=SRWLIB_VirtBL_CHX_01.py; stack:
Traceback (most recent call last):
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/importer.py", line 122, in import_python
    arguments=arguments,
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/importer.py", line 64, in __init__
    self.data = _parsed_dict(self.var_param, self.optics)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/importer.py", line 545, in _parsed_dict
    beamline_elements = _get_beamline(op.arOpt, v.op_r)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/importer.py", line 411, in _get_beamline
    positions[i]['dist_source']
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/importer.py", line 280, in _beamline_element
    raise Exception('Element type <{}> does not exist.'.format(elem_type))
Exception: Element type <cyl_fiber> does not exist.
```

**Comment by mrakitin** [comment]
Implemented in f9c3b01. Test added in a607523.

**Comment by mrakitin** [comment]
`focusingPlane` parameter left in the JS code. Need to rename it to `focalPlane` as it's used wherever in the code.

**Comment by mrakitin** [comment]
Fixed in 74b7099.
